pkg/trace/pb/decoder.go

```diff
@@ -26,13 +48,20 @@ func parseString(dc *msgp.Reader) (string, error) {
 		if err != nil {
 			return "", err
 		}
-		return msgp.UnsafeString(i), nil
+		if utf8.Valid(i) {
+			// TODO this really isn't safe
```

Comment:
I'd say we remove this or add details about the work needed to be done. 
```suggestion
			// TODO(x) this really isn't safe
```
`x` is usually the Github username of the person meant to address this `TODO`, but in case it's ok for anyone we can leave it at `x` as long as there is a clear explanation of what that person should do.

Reply:
OK. I'm concerned because it's a pointer to a `[]byte` of which the lifecycle isn't owned. I can add more details.